app/api/v2/views.py

```diff
@@ -0,0 +1,52 @@
+from flask_restful import Resource
+from flask import make_response, jsonify, request
+from ...utilities.validation_functions import check_for_space
+from .models import OrderParcel
+
+order = OrderParcel()
+
+class ParcelList(Resource):
+    """
+    class for Create order and retrieve list of orders API endpoints
+    """
+    def post(self):
+        """
+        post method to add new order to list of orders
+        """
+        data = request.get_json()
+        item = data['item']
+        pickup = data['pickup']
+        dest = data['dest']
+        pricing = data['pricing']
+        author = data['user_id']
+
+        if check_for_space(item):
```

Comment:
Let's get rid of the nested `ifs`. Check for the NOT condition. Then return the response.